docs/dptp-triage-sop/openshift-priv.md

```diff
@@ -0,0 +1,45 @@
+# `openshift-priv` Organization
+
+## Unrelated histories
+
+A force-push in one of the repositories which has a counterpart in
+`openshfit-priv` will cause the job to fail to synchronize the repositories,
+e.g.:
+
+```
+time="2022-11-01T08:23:40Z" level=warning msg="error occurred while fetching remote and merge" branch=release-4.13 destination=openshift-priv/cluster-api-provider-ibmcloud@release-4.13 error="[failed to merge openshift-cluster-api-provider-ibmcloud/release-4.13: failed with 128 exit-code: fatal: refusing to merge unrelated histories\n, failed to perform merge --abort: failed with 128 exit-code: fatal: There is no merge to abort (MERGE_HEAD missing).\n]" local-repo=/tmp/1970879496/openshift/cluster-api-provider-ibmcloud org=openshift repo=cluster-api-provider-ibmcloud source=openshift/cluster-api-provider-ibmcloud@release-4.13 source-file=openshift-cluster-api-provider-ibmcloud-release-4.13.yaml variant=
+```
+https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-openshift-release-private-org-sync/1587355868737835008
+
+Because merge conflicts are [ignored][private_org_sync_readme] by
+`private-org-sync`, this will not cause the synchronization job to fail;
+instead, the two repositories will silently diverge.  Eventually, this will
+result in support requests when the owners notice their repository has not been
+updated.
+
+Manual intervention is required to reconcile the two repositories.  For example,
```

Comment:
> Manual intervention is required

I would like to avoid it as much as possible. Should we open a card to understand if we can recognize what repos need to be manually renconciled and therefore automate the whole thing?

Reply:
How would they be reconciled?

Reply:
> How would they be reconciled?
> > open a card to understand

I can understand we expect two repositories to diverge once CVEs come into play and therefore not to trigger any warnings that would otherwise produce only noise.
On the other way around, it would useful to fire something when they do diverge because the non-priv one was forcibly rebased.

Reply:
From https://github.com/openshift/ci-tools/pull/3134:

> While it's debatable whether all the silent failures we currently have are
> preferable to the problem described in DPTP-1426, this at least documents the
> behavior to save future-us some time.

What you describe would require its own separate effort and investigation, while this pair of pull requests have the objective of documenting exactly what our _current_ policy is.